src/components/features/team-cards/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import IconButton from "../../ui/iconbutton";
+import Section from "../../ui/section";
+import FeatureHeading from "../../ui/feature-heading";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine